contributors/devel/sig-architecture/import-boss.md

```diff
@@ -0,0 +1,69 @@
+# Using import-boss to enforce package import restrictions
+
+This document shows a way of enforcing package import restrictions using the `import-boss` tool.
+
+## Background
+
+The import-boss check runs against all pull requests submitted against the k/k repository. There are currently a good
+number of import restriction files, called `.import-restrictions` in the repository. The import restrictions files are
+specified in YAML format.
+
+## Rules
```

Comment:
also here describe in one sentence what these do: "Rules specify which Golang `import "<import path>"` are permitted"